docker/Dockerfile

```diff
@@ -61,6 +60,6 @@ USER $MAMBA_USER
 RUN ${CONDA_RUN} bash -c 'curl -L https://fly.io/install.sh | sh'
 ENV PATH="${CONTAINER_HOME}/.fly/bin:$PATH"
 
-
+WORKDIR ${PUDL_REPO}
```

Comment:
The GCP ETL script expects to be run from within the repository, so make sure that's where we end up at the end of the build.